stable/appmesh-prometheus/README.md

```diff
@@ -21,6 +21,29 @@ helm upgrade -i appmesh-prometheus eks/appmesh-prometheus \
 --namespace appmesh-system
 ```
 
+Enable EBS CSI Driver:  
```

Comment:
I see in the chart that use of a persistent volume is controlled by persistentVolumeClaim.claimName. Is this section only needed if that is configured?

Reply:
I think so. Because the original README creates a PersistentVolumeClaim, the PVC is pending without the EBS driver, but there may be other ways to mount it, such as NFS.